src/main/java/teammates/logic/core/ProfilesLogic.java

```diff
@@ -45,4 +47,14 @@ public void updateStudentProfilePicture(String googleId, String newPictureKey) t
         profilesDb.updateStudentProfilePicture(googleId, newPictureKey);
     }
 
+    /**
+     * Gets all student profiles in the Datastore.
```

Comment:
I think student profiles are more intuitive as the entity is called `StudentProfile`.